src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -120,28 +128,25 @@ internal static ImageLoadingEstimator LoadImages(this TransformsCatalog catalog,
 
-        /// <include file='doc.xml' path='doc/members/member[@name="ImagePixelExtractingEstimator"]/*' />
+        /// <summary>
+        /// Create a <see cref="ImagePixelExtractingEstimator"/>, which exctracts pixel values from the data specified in column: <see cref="ImagePixelExtractingEstimator.ColumnOptions.InputColumnName"/>
+        /// to a new column: <see cref="ImagePixelExtractingEstimator.ColumnOptions.Name"/>.
+        /// </summary>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="columnOptions">The <see cref="ImagePixelExtractingEstimator.ColumnOptions"/> describing how the transform handles each image pixel extraction output input column pair.</param>
         [BestFriend]
         internal static ImagePixelExtractingEstimator ExtractPixels(this TransformsCatalog catalog, params ImagePixelExtractingEstimator.ColumnOptions[] columnOptions)
             => new ImagePixelExtractingEstimator(CatalogUtils.GetEnvironment(catalog), columnOptions);
 
         /// <summary>
-        /// Resizes the images to a new width and height.
+        /// Create a <see cref="ImageResizingEstimator"/>, which resize the image from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
-        /// <remarks>
-        /// In image processing pipelines, often machine learning practitioner make use of<a href= "https://blogs.msdn.microsoft.com/mlserver/2017/04/12/image-featurization-with-a-pre-trained-deep-neural-network-model/">
-        /// pre-trained DNN featurizers</a> to extract features for usage in the machine learning algorithms.
-        /// Those pre-trained models have a defined width and height for their input images, so often, after getting loaded, the images will need to get resized before
-        /// further processing.
-        /// The new width and height can be specified in the <paramref name="imageWidth"/> and <paramref name="imageHeight"/>
-        /// <seealso cref = "ImageEstimatorsCatalog" />
-        /// <seealso cref= "ImageLoadingEstimator" />
-        /// </remarks >
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be the same as that of the input column.</param>
+        /// <param name="inputColumnName">Name of the column to copy the data from.
```

Comment:
>Name of the column to copy the data from. [](start = 42, length = 41)

fix me :)

Reply:
I can do what, but who gonna fix me?

---
In reply to: [276906121](https://github.com/dotnet/machinelearning/pull/3376#discussion_r276906121) [](ancestors = 276906121)